packages/fiori/src/themes/sap_horizon/ShellBar-parameters.css

```
@@ -1,10 +1,31 @@
 :root {
+	--_ui5_shellbar_root_height: 3.25rem;
 	--_ui5_shellbar_logo_outline_color: var(--sapContent_FocusColor);
 	--_ui5_shellbar_logo_outline: var(--sapContent_FocusWidth) var(--sapContent_FocusStyle) var(--_ui5_shellbar_logo_outline_color);
 	--_ui5_shellbar_outline_offset: -0.125rem;
+	--_ui5_shellbar_button_border: none;
+	--_ui5_shellbar_button_border_radius: var(--sapButton_BorderCornerRadius);
+	--_ui5_shellbar_logo_border_radius: 0.5rem;
 	--_ui5_shellbar_button_box_shadow: inset var(--sapContent_Interaction_Shadow);

```

Remove the inset, looks closer to the target spec without it:

```suggestion
	--_ui5_shellbar_button_box_shadow: var(--sapContent_Interaction_Shadow);

```